### PR TITLE
Adding string permutation and combinations [Python]

### PR DESCRIPTION
**Pull request by darkrishabh**

### CODE REVIEW

File: python/string/stringPermutation.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    unique_string = ''.join([j for i, j in enumerate(string) if j not in string[:i]])
+    for s in string:
+        if s in freq_map:
+            freq_map[s] += 1
+        else:
+            freq_map[s] = 1
+    return [freq_map[s] for s in unique_string], unique_string
+
+
+original_string = "AABC"
```
Comment (mission-peace) [inline python/string/stringPermutation.py]: Can you put all this under if condition 
if __name__ == '__main__':

File: python/string/stringCombinations.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    unique_string = ''.join([j for i, j in enumerate(string) if j not in string[:i]])
+    for s in string:
+        if s in freq_map:
+            freq_map[s] += 1
+        else:
+            freq_map[s] = 1
+    return [freq_map[s] for s in unique_string], unique_string
+
+
+original_string = "AABC"
```
Comment (mission-peace) [inline python/string/stringCombinations.py]: Please put this under block 
if __name__ == '__main__':

**Comment by orsenthil** [comment]
One more suggestion. do a squash commit like this and push it to this branch.

```
git rebase -i origin/master

```
* Set your `$EDITOR=vim`  # or something relevant
* Use squash for bottom n-1 commits instead of pick.
* Save File, this will rewrite history and you could give a single commit message. 

and then do a 
```
git push origin your_branch_name -f
```

This will squash all the changes to a single commit and is good to merge.

**Comment by mission-peace** [comment]
No response. Closing.

**Pull request closed without merging**